src/test/java/com/snowflake/kafka/connector/builder/SinkRecordBuilder.java

```diff
@@ -50,4 +54,10 @@ public SinkRecordBuilder withOffset(long offset) {
     this.offset = offset;
     return this;
   }
+
+  public SinkRecordBuilder withTimestamp(long timestamp, TimestampType timestampType) {
```

Comment:
probably overkill, but should we assert on NO_TIMESTAMP_TYPE ? it seems like this value wouldn't be allowed for non-null timestamp value.